app-full-microprofile/threshold.properties

```diff
@@ -2,5 +2,5 @@ linux.jvm.time.to.first.ok.request.threshold.ms=2600
 linux.jvm.RSS.threshold.kB=550000
 linux.native.time.to.first.ok.request.threshold.ms=50
 linux.native.RSS.threshold.kB=120000
```

Comment:
Thresholds can be defined on command line too - https://github.com/quarkus-qe/quarkus-startstop#thresholds

We do not activelly run on Windows, so making direct adjustments seems OK for now